protobuf/v1/pool.proto

```diff
@@ -36,6 +36,9 @@ message CreatePoolRequest {
   google.protobuf.StringValue uuid = 2; // optional uuid for the pool to be created
   repeated string disks = 3; // disk device paths or URIs to be claimed by the pool
   PoolType pooltype = 4;     // type of the pool
+  // Default cluster size is 4MiB (4 * 1024 * 1024), input cluster size must be multiple of 1MiB, 
+  // else default cluster size will be considered.
+  optional uint32 cluster_size = 5;
```

Comment:
can we also report cluster_size on Pool message below?

Reply:
> can we also report cluster_size on Pool message below?

Sure.. Added..

Reply:
In the `ReplicaSpaceUsage` already have the cluster_size information which is return as part of ListReplica.

Reply:
oh cool!